packages/hash/api/src/db/postgres/link/getEntityOutgoingLinks.ts

```diff
@@ -36,3 +56,59 @@ export const getEntityOutgoingLinks = async (
 
   return rows.map(mapDBLinkRowToDBLink);
 };
+
+const outgoingLinkAggregationQuery = sql`select array_agg(destination_entity_id) as outgoing_entity_ids from links
+where a.entity_id = source_entity_id
+group by a.entity_id`;
+
+/**
+ * Get the latest version of all entities of a given type and their outgoing link ids with a matching entity type.
```

Comment:
It doesn't look like "with a matching entity type" belongs here - it's just getting entities of a given type and their outgoing link ids, no?

Reply:
You're right. I think I removed that part of the query when i switched to a lateral left join. Will add it back, as it would otherwise returns links that might not be relevant for page trees